README.md

```
@@ -1 +1,59 @@
+---
+
+## 📍 맛집 추천
+
+
+---
+
+## 📜 팀 규칙
+- [Team Rules](docs/team-rules.md)
+
+## 저장소 구조
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)